### PR TITLE
Minor config fixes

### PR DESCRIPTION
**Pull request by romidane**
This solves https://github.com/sky-uk/blackjack-toolbelt/issues/27 , https://github.com/sky-uk/blackjack-toolbelt/issues/26 and https://github.com/sky-uk/blackjack-toolbelt/issues/25

### CODE REVIEW

File: src/preview-server/bootstrap.js [hunk 1]
```diff
@@ -6,10 +6,11 @@ import logger from '../util/logger';
 import {
   BLACKJACK_HOME,
   WORKING_DIR,
-  COMPONENT_CONF
+  COMPONENT_CONF,
+  SERVER_PORT
 } from '../constants';
 
-const SERVER_PORT = process.env.PORT || 3000;
+const PORT = process.env.PORT || SERVER_PORT;
```
Comment (joshnesbitt) [inline src/preview-server/bootstrap.js]: Maybe move the `process.env.PORT` check to `constants.js` so it's contained in one place?

File: src/constants.js [hunk 2]
```diff
@@ -11,5 +11,6 @@ module.exports = {
   KARMA_BIN: `${MODULES_DIR}/karma/bin/karma`,
   KARMA_CONF: `${BLACKJACK_DIR}/src/vendor-integration/karma/config.js`,
   MOCHA_BIN: `${MODULES_DIR}/mocha/bin/mocha`,
-  TEST_GLOB: `${WORKING_DIR}/test/**/*.js`
+  TEST_GLOB: `${WORKING_DIR}/test/**/*.js`,
+  SERVER_PORT: 9876
```
Comment (joshnesbitt) [inline src/constants.js]: Maybe call this `PREVIEW_SERVER_PORT` for clarity.

File: src/preview-server/bootstrap.js [hunk 3]
```diff
@@ -85,6 +99,6 @@ app.get('/', (req, res) => {
   res.render('index', { template: '', brand: '', component: '' });
 });
 
-logger.info(`Starting server on http://localhost:${SERVER_PORT}`);
+logger.info(`Launching server and listening for changes on http://localhost:${PORT}.`);
```
Comment (joshnesbitt) [inline src/preview-server/bootstrap.js]: 👍

File: src/preview-server/bootstrap.js (hunk 1 above)
Comment (romidane) [inline src/preview-server/bootstrap.js]: Good point 👍

File: src/constants.js (hunk 2 above)
Comment (romidane) [inline src/constants.js]: 👍

**Comment by romidane** [comment]
Updated

**Comment by joshnesbitt** [comment]
Nice work :shipit:

**Pull request merged**